src/site/SearchResultSet.php

```diff
@@ -0,0 +1,52 @@
+<?hh // strict
+
+namespace HHVM\UserDocumentation;
+
+final class SearchResultSet {
+    private Map<string, string> $classes = Map {};
+    private Map<string, string> $functions = Map {};
+    private Map<string, string> $interfaces = Map {};
+    private Map<string, string> $hhvm_guides = Map {};
+    private Map<string, string> $hack_guides = Map {};
```

Comment:
could be worth doing some typedefs - eg turn this into `Map<DocumentedAPIDefinitionName, SiteURLPath>` ?